pegasus/sites.v3/code.org/public/js/workshop-search.js

```diff
@@ -23,83 +25,89 @@ function initializeMap() {
 }
 
 function loadWorkshops() {
-  var params = [];
+  markerClusterer = new MarkerClusterer(gmap, [],
+    {imagePath: 'https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/m'}
```

Comment:
D'oh! Thanks. I apparently didn't read this [part](https://developers.google.com/maps/documentation/javascript/marker-clustering#adding-a-marker-clusterer)

> Get the marker clustering library and images from GitHub, and store them on a server accessible to your app. 

Will update...